cylc/uiserver/config_defaults.py

```diff
@@ -46,3 +47,7 @@
 c.JupyterHub.cookie_secret_file = f'{RUNTIME_PATH / "cookie_secret"}'
 c.JupyterHub.db_url = f'{RUNTIME_PATH / "jupyterhub.sqlite"}'
 c.ConfigurableHTTPProxy.pid_file = f'{RUNTIME_PATH / "jupyterhub-proxy.pid"}'
+
+# Ensure env variable CYLC_SITE_CONF_PATH is passed to UI Server environment
+c.Spawner.environment = {
+    'CYLC_SITE_CONF_PATH': f'{os.getenv("CYLC_SITE_CONF_PATH")}'}
```

Comment:
Hmmm, I think there's a simpler way to tell JupyterHub what variables must be passed down to the UI Server. Will search their docs...

Reply:
I went with this method as it was recommended on an [issue](https://github.com/jupyterhub/the-littlest-jupyterhub/issues/308). 
From reading the docs we could also use : `c.Spawner.env_keep.append('CYLC_SITE_CONF_PATH')` which may be a bit nicer.

Reply:
Ah, I think that's the one I was thinking. If both work the same way, in local and remote environments, then either way is fine by me.